dts/xtensa/espressif/esp32s3/esp32s3_common.dtsi

```diff
@@ -336,6 +336,15 @@
 			status = "disabled";
 		};
 
+		lcd_cam: lcd_cam@60041000 {
+			compatible = "espressif,esp32-lcd-cam";
+			reg = <0x60041000 DT_SIZE_K(4)>;
+			clocks = <&rtc ESP32_LCD_CAM_MODULE>;
+			interrupts = <LCD_CAM_INTR_SOURCE IRQ_DEFAULT_PRIORITY 0>;
```

Comment:
I'd rather leave as is so we got the proper info described here even if we don't use it now on the driver. Later it can be useful when extending the driver.